tests/strictequal.test.js

```diff
@@ -0,0 +1,52 @@
+const describe = require('mocha').describe
+const it = require('mocha').it
+const expect = require('chai').expect
+const strictEqual = require('../strictequal')
+
+describe('test 1 fail on parameter types types', () => {
+  describe('test 1 parameter are different types', () => {
+    it('test 1 types differ', () => {
```

Comment:
Your test descriptions should fully describe what is happening and expected. Something like `returns false when the parameters are the same value but different types` would be a lot clearer.